src/rp2_common/pico_audio_i2s/audio_i2s.c

```diff
@@ -52,10 +52,18 @@ const audio_format_t *audio_i2s_setup(const audio_format_t *intended_audio_forma
     gpio_set_function(config->clock_pin_base, func);
     gpio_set_function(config->clock_pin_base + 1, func);
 
+#if PICO_PIO_VERSION > 0
```

Comment:
I think i'd rather guard this with `NUM_GPIOS > 32`

There'll be a bit of a pattern here which is to minimize additional runtime code that is linked in either in RP2040 or RP2350A

Reply:
scratch that; even better, use `#if PICO_PIO_USE_GPIO_BASE`

Reply:
Done.